de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/browser/BrowserAction.java

```diff
@@ -0,0 +1,35 @@
+package de.fu_berlin.inf.dpp.whiteboard.ui.browser;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class BrowserAction {
+
+    public class ActionTypes {
+        public final static String NEW = "NEW";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reorder the modifiers to comply with the Java Language Specification. ([squid:ModifiersOrderCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AModifiersOrderCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=250&issues=AWY_InU0nzfF1D4JrUtU&open=AWY_InU0nzfF1D4JrUtU)

[ISSUE_KEY]: AWY_InU0nzfF1D4JrUtU